packages/sync-utils/src/sanity/syncSanityDocument.ts

```diff
@@ -149,7 +151,8 @@ export const createSyncSanityDocument = (
   const syncItem = async (
     item: Product | Collection
   ): Promise<SyncOperation> => {
-    const docInfo = prepareDocument(item)
+    const { shopName } = shopifyClient
+    const docInfo = prepareDocument({ ...item, shopName })
     const existingDoc = await getSanityDocByShopifyId(item.id)
 
     /* If the document exists and is up to date, skip */
```

Comment:
could you add `shopName` to the list of keys below? This will make sure all existing documents get updated when running a new sync.

Reply:
Ok let's check if it's better now